js/editPairForm.js

```diff
@@ -385,14 +389,15 @@ $(document).ready(function () {
           // startsCapital: true
         },
         d_addr1: {
-          supernumeric: true
+          //supernumeric: true
         },
         d_addr2: {
           supernumeric: true
         },
         d_city: {
-          nowhitespace: true,
-          lettersonly: true,
+          lettersandspace:true
```

Comment:
Instead of creating a new function(i.e, `lettersandspace`). Can you reuse the `supernumeric` method here?